packages/web-components/src/components/cbp-form-field-wrapper/cbp-form-field-wrapper.stories.tsx

```diff
@@ -74,8 +76,119 @@ InputWithOverlays.args = {
   value: '',
 };
 
-// TechDebt: needs an event listener to swap the button's icon and input type (password | text)
+
+
+const NumericCounterTemplate = ({ label, description, inputType,  overlayStart, overlayEnd, fieldId, error, readonly, disabled, value, context, sx }) => {
+
+  // Ideally, this should be placed on the button component itself, not the document; but the event bubbles, so it works here.
+  document.addEventListener('buttonClick', function(e) {
+    const buttonComponent = e.target as HTMLCbpButtonElement;
+    const button = buttonComponent.querySelector('button') as HTMLButtonElement;
+    const input: HTMLInputElement = document.querySelector(`#${fieldId}`) || document.querySelector('input');
+    const step:number = Number(input.getAttribute('step')) || 1;
+
+    //console.log({e});
```

Comment:
I assume this would be nice for testing later but wanted to make sure you meant to leave it